db/data.py

```diff
@@ -520,6 +520,71 @@ def load_many_high_level(recordings):
         return dict(recordings_info)
 
 
+def load_many_select_features(recordings, features):
+    """Load select lowlevel features for multiple recordings.
+    
+    Args:
+        recordings: A list of tuples (mbid, offset).
+        features: Contains information about the
+        select features, a list of tuples of the form: 
+            [(<feature_path>, <alias>, <default_type>), ...]
+
+            <feature_path> is a string holding the path to a feature:
+            "llj.data->feature_name"
+
+            <alias> is a string alias for a feature:
+            "lowlevel.feature_name"
+            
+            <default_type> is the type to which a feature value will default
+            if it is non-existent: None, {}, or [].
+
+    Returns:
+        {"mbid-1": {"offset-1": {lowlevel document},
+                    ...
+                    "offset-n": {lowlevel document},
+         ...
+         "mbid-n": {"offset-1": {lowlevel document}}
+        } 
+    """
+    # Build string of select features with aliases
+    feature_string = features[0][0] + ' AS "' + features[0][1] + '", '
+    for path, alias, _ in features[1:len(features)-1]:
+        feature_string += path + ' AS "' +  alias + '", '
+    feature_string += features[len(features)-1][0] + ' AS "' + features[len(features)-1][1] + '"'
+
+    with db.engine.connect() as connection:
+        query = text("""
+            SELECT ll.gid::text
+                 , ll.submission_offset::text
+                 , %(features)s
+              FROM lowlevel ll
+              JOIN lowlevel_json llj
+                ON ll.id = llj.id
+             WHERE (ll.gid, ll.submission_offset)
+                IN :recordings
+        """ % {'features': feature_string})
+        result = connection.execute(query, {'recordings': tuple(recordings)})
+
+        recordings_info = defaultdict(dict)
+        for row in result.fetchall():
+            # Build dictionary of feature columns
+            data = defaultdict(dict)
```

Comment:
I would break this indented section into a separate function (input row, output data), and then have some tests for it